Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java

```diff
@@ -36,16 +36,8 @@
 import com.privatejgoodies.forms.factories.CC;
 import com.privatejgoodies.forms.layout.CellConstraints;
 import com.privatejgoodies.forms.layout.FormLayout;
-import java.awt.Canvas;
-import java.awt.Color;
-import java.awt.Dimension;
-import java.awt.Font;
-import java.awt.FontMetrics;
-import java.awt.GridBagConstraints;
-import java.awt.GridBagLayout;
-import java.awt.Insets;
-import java.awt.Point;
-import java.awt.Rectangle;
+
+import java.awt.*;
```

Comment:
Please follow the project standards for imports.  Wildcard or package level imports are not supported; each class use should be explicitly enumerated.  Thanks